source/conf.py

```diff
@@ -5,8 +5,8 @@
 project = 'NGINX Unit'
 author = 'NGINX, Inc.'
 copyright = '2017-2024'
-version = '1.32.1'
-release_date = 'Mar 26, 2024'
+version = '1.33.0'
+release_date = 'Aug 29, 2024'
```

Comment:
```suggestion
release_date = 'Aug 29, 2024'
```
This needs to be updated before merge